bin/templates/scripts/cordova/lib/build.js

```diff
@@ -88,25 +89,61 @@ module.exports.run = function (buildOpts) {
         events.emit('log','\tConfiguration: ' + configuration);
         events.emit('log','\tPlatform: ' + (buildOpts.device ? 'device' : 'emulator'));
 
-        var xcodebuildArgs = getXcodeArgs(projectName, projectPath, configuration, buildOpts.device);
+        var xcodebuildArgs = getXcodeBuildArgs(projectName, projectPath, configuration, buildOpts.device);
         return spawn('xcodebuild', xcodebuildArgs, projectPath);
     }).then(function () {
         if (!buildOpts.device || buildOpts.noSign) {
             return;
         }
+
+        var exportOptions = {'compileBitcode': false};
+
+        if (buildOpts.packageType) {
+            exportOptions.method = buildOpts.packageType;
```

Comment:
Yeah we should. I just tested it, and it works with that default. Must be an xcodebuild bug.